xc7/archs/artix7/tiles/CMakeLists.txt

```diff
@@ -2,8 +2,18 @@ add_subdirectory(bram_l)
 add_subdirectory(bram_r)
 add_subdirectory(slicel)
 add_subdirectory(slicem)
-add_subdirectory(riopad_m)
 add_subdirectory(clk_bufg_top_r)
 add_subdirectory(clk_bufg_bot_r)
 add_subdirectory(clk_hrow_top_r)
 add_subdirectory(clk_hrow_bot_r)
+
```

Comment:
This seems super hacky....

Don't know a better solution however....

Reply:
How is it hacky?  I actually think it is correct.  What specifically seems wrong?